### PR TITLE
Update phone-number-management-for-australia.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/2133
Added note under steps.

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-phone-numbers-for-your-organization/phone-number-management-for-australia.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ For more information, see  [Manage phone numbers for your organization](manage-p
 |Get phone numbers transferred to Office 365 for users, auto attendants and call queues.  <br/> |Please contact [Telstra Calling for Office 365](https://aka.ms/TelstraVoicePlan) to transfer phone numbers. They will then add those numbers to your organization.  <br/> |
 |Get phone numbers transferred to Office 365 for use with Audio Conferencing.  |[Please contact us](mailto:ptnapac@microsoft.com) |
 |Change emergency addressing assigned to users that have a Call Plan phone number. |Please contact [Telstra Calling for Office 365](https://aka.ms/TelstraVoicePlan) to make emergency address changes. They can update this information for you.|
-|Change a user number to a service number. |This type service isn't supported in Australia.
+|Change a user number to a service number. |It's supported when using Telstra Calling for Office 365.
```
Comment (nenonix) [inline Teams/manage-phone-numbers-for-your-organization/phone-number-management-for-australia.md]:
```suggestion
|Change a user number to a service number. |Supported when using Telstra Calling for Office 365.
```

**Comment by scanum** [comment]
@CarolynRowe Changes look correct. Could you please review and merge? Thanks!

**Comment by CarolynRowe** [comment]
@tonysmit, are you OK with this change?

**Pull request merged**